BT.json

```diff
@@ -664,7 +664,7 @@
 			"wowID":"32497"
 		},
 		"The Skull of Gul'dan":{
-			"prio":["Coxxy(1)","Demondred(1)","Kitboga(1)","Killmcpickle(2)","Warlocks > Boomkin/Ele > Mages > Spriest","EP/GP"],
+			"prio":["Coxxy(1)","Demondred(1)","Kitboga(1)","Microdoses(1)","Rubski(2)","Warlocks > Boomkin/Ele > Mages > Spriest","EP/GP"],
```

Comment:
micro still 3rd on ZD?

Reply:
Yeah, skull hasn't dropped yet and it might be a while before Micro gets a shot at it; I still want to reward him with something big this phase.